src/BulletDynamics/Dynamics/btRigidBody.cpp

```diff
@@ -146,10 +151,9 @@ void btRigidBody::applyDamping(btScalar timeStep)
 	//On new damping: see discussion/issue report here: http://code.google.com/p/bullet/issues/detail?id=74
 	//todo: do some performance comparisons (but other parts of the engine are probably bottleneck anyway
 
-//#define USE_OLD_DAMPING_METHOD 1
-#ifdef USE_OLD_DAMPING_METHOD
-	m_linearVelocity *= GEN_clamped((btScalar(1.) - timeStep * m_linearDamping), (btScalar)btScalar(0.0), (btScalar)btScalar(1.0));
-	m_angularVelocity *= GEN_clamped((btScalar(1.) - timeStep * m_angularDamping), (btScalar)btScalar(0.0), (btScalar)btScalar(1.0));
+#ifdef BT_USE_OLD_DAMPING_METHOD
+	m_linearVelocity *= btClamped((btScalar(1.0) - timeStep * m_linearDamping), btScalar(0.0), btScalar(1.0));
+	m_angularVelocity *= btClamped((btScalar(1.0) - timeStep * m_angularDamping), btScalar(0.0), btScalar(1.0));
```

Comment:
You can just use max here, because it can't be more than 1, so we save one if condition.

```cpp
m_linearVelocity *= btMax(btScalar(1.0) - timeStep * m_linearDamping, btScalar(0.0));
m_angularVelocity *= btMax(btScalar(1.0) - timeStep * m_angularDamping, btScalar(0.0));
```

Reply:
Thanks, I've just pushed this change.

Reply:
Super!